doc/filesize.rst

```diff
@@ -0,0 +1,72 @@
+The Filesize Extension
+======================
+
+The *Filesize* extension provides the ``filesize`` filter.
+
+You need to register this extension before using the ``filesize`` filter::
+
+    $twig->addExtension(new Twig_Extensions_Extension_Filesize());
+
+``filesize``
```

Comment:
I don't think the name is good enough. It's more generic than that and not limited to *file* sizes (think HTTP response content size for another good usage example). Also, this extension can probably do the same for other units like time (s, ms, ...). So, I would prefer a generic **unit** extension rather than a very specific **filesize** one.